adagrams/game.py

```diff
@@ -1,13 +1,115 @@
 from random import randint
 
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    #make a new list named pool_list
+    pool_list = []
+    for key, value in LETTER_POOL.items():
+        for i in range(0,value):
+            pool_list.append(key)
```

Comment:
We should avoid creating variables that we will not use to reduce the number of places where bugs could possibly happen. Since `i` is not used, we could instead write this loop as:
```suggestion
        for _ in range(0, value):
            pool_list.append(key)
```

I also strongly recommend avoiding single letter variable names in your code. If `i` represents an `index` then we should use the full name to be as clear as possible with our meaning.